src/event.js

```diff
@@ -5,10 +5,11 @@ define( [
 	"./var/rnothtmlwhite",
 	"./var/slice",
 	"./data/var/dataPriv",
+	"./manipulation/var/rcheckableType",
```

Comment:
@gibson042 thanks, got it. Now I'm new to the project so sorry if the question might seem silly...
You're basically asking to change to
"./var/rcheckableType",

And that would also mean I'd need to change the location of rcheckabletype in the file structure too since it's currently inside the manipulation folder and not on the top level var? And that would also require a change of the "./manipulation/var/rcheckableType" line on the other files that require it as well, correct?

thanks again